src/Nethermind/Nethermind.Specs.Test/ChainSpecStyle/ChainSpecBasedSpecProviderTests.cs

```diff
@@ -166,8 +166,9 @@ public void Holesky_loads_properly()
         Assert.That(provider.ChainId, Is.EqualTo(BlockchainIds.Holesky));
         Assert.That(provider.NetworkId, Is.EqualTo(BlockchainIds.Holesky));
 
-        GetTransitionTimestamps(chainSpec.Parameters).Should().AllSatisfy(
-            t => ValidateSlotByTimestamp(t, HoleskySpecProvider.GenesisTimestamp).Should().BeTrue());
+        // because genesis time for holesky is set 5 minutes before the launch of the chain. this test fails.
+        //GetTransitionTimestamps(chainSpec.Parameters).Should().AllSatisfy(
```

Comment:
Why doesn't the timestamp follow the 8192 rules?

Reply:
because the timestamp of holesky genesis is set to be 5 before the chain starts

Reply:
oh, I see the comment - thx